Update Documentation with Pydata Sphinx Theme, and more (#523)
rapidsai/cuspatial: docs/source/api_docs/trajectory.rst

Trajectory
----------

> Trajectory functions make it easy to identify and group trajectories from point data.

Reviewer: ```suggestion
Functions for identifying and grouping trajectories from point data.
```
